lib/index.js

```diff
@@ -70,7 +70,7 @@ module.exports = function(options) {
   options.getApiVersion = options.getApiVersion || noop;
 
   // logBody option
-  options.logBody = options.logBody == false ? false : true;
+  options.logBody = (options.logBody !== false);
```

Comment:
This has the same problem where `undefined` is treated as `true`. The `logBody` value should default to `true`